app/models/service_request.rb

```diff
@@ -84,6 +85,11 @@ def self.filter_by_search_311(params)
     requests
   end
 
+  def public_servant_description_validation
+      if public_servant_id == nil
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer the use of the `nil?` predicate.